htdocs/core/class/commonobject.class.php

```diff
@@ -3237,6 +3236,10 @@ public function fetchObjectLinked($sourceid = null, $sourcetype = '', $targetid
 		$targetid = (!empty($targetid) ? $targetid : $this->id);
 		$sourcetype = (!empty($sourcetype) ? $sourcetype : $this->element);
 		$targettype = (!empty($targettype) ? $targettype : $this->element);
+		if (!empty($this->module)) {
+		    $sourcetype=$this->module."@$sourcetype";
```

Comment:
The module name must be after the @ not before to match other coding convention
object@module